src/test/tests.ts

```diff
@@ -1,3 +1,6 @@
+// chai style expect().to.be.true violates no-unused-expression
+/* tslint:disable:no-unused-expression */
```

Comment:
Normally I'd argue for applying this line by line only, but there are probably lots of these sprinkled throughout the file, so this is okay.